milestone3/frontend/src/components/Homepage/Homepage.tsx

```diff
@@ -0,0 +1,33 @@
+import { Recipe } from "../../types"
+import { useEffect, useState } from "react"
+import RecipePreview from "../RecipePreview/RecipePreview"
+
+export default function Homepage() {
+  const [recipes, setRecipes] = useState<Recipe[]>([])
+  useEffect(() => {
+    async function fetchRecipes() {
+      const response = await fetch("https://bootcamp-milestone-4.onrender.com/recipe")
+      const recipes = await response.json()
+      setRecipes(recipes)
+    }
+    fetchRecipes()
```

Comment:
A .then().catch() chain would be more efficient here, but if you want to keep it like this you should wrap your awaits in a try/catch block so that you can catch and print any errors with your api calls, can save you a lot of time if a bug comes up later.

Reply:
Thanks, fixed this